torntools/scripts/content/items/ttItems.js

```diff
@@ -665,3 +669,97 @@ function addItemListener() {
 
 	doc.find("head").appendChild(script);
 }
+
+function showMissingPlushies() {
+	if (settings.pages.items.show_missing_plushies) {
+		let plushieSet = {
+			"Sheep Plushie": 186,
+			"Teddy Bear Plushie": 187,
+			"Kitten Plushie": 215,
+			"Jaguar Plushie": 258,
+			"Wolverine Plushie": 261,
+			"Nessie Plushie": 266,
+			"Red Fox Plushie": 268,
+			"Monkey Plushie": 269,
+			"Chamois Plushie": 273,
+			"Panda Plushie": 274,
+			"Lion Plushie": 281,
+			"Camel Plushie": 384,
+			"Stingray Plushie": 618,
+		};
+		let plushiesAvailable = userdata.inventory
+			.filter((x) => x.type === "Plushie")
+			.map((x) => x.name);
+		if (plushiesAvailable.length > 0) {
+			let neededPlushies = Object.keys(plushieSet).filter(
+				(x) => !plushiesAvailable.includes(x)
+			);
+			let neededPlushiesDiv = doc.new({
+				type: "div",
+				id: "tt-needed-plushies-div",
+			});
+			for (let plushieNumber in neededPlushies) {
```

Comment:
Instead of using `for (let x in y)` you should use `for (let x of y)` here because you don't really need the index.

```
let y = ['a', 'b', 'c'];
for (let x in y) {}  // loops through indexes: 0,1,3
for (let x of y) {}  // loops through items themselves: 'a', 'b', 'c'
```

Reply:
Done.